index.html

```diff
@@ -876,14 +927,21 @@
         <pre class="idl">
           interface PresentationAvailability : EventTarget {
             readonly attribute boolean value;
+
             attribute EventHandler onchange;
           };
-
+        
 </pre>
         <p>
           A <a><code>PresentationAvailability</code></a> object is associated
-          with a <a>presentation display</a> and represents its
-          <dfn>presentation display availability</dfn>.
+          with available <a>presentation displays</a> and represents the
+          <dfn>presentation display availability</dfn> for a presentation
+          request. The <a><code>PresentationAvailability</code></a> object
+          SHOULD be implemented in a <a>controlling browsing context</a> by a
+          <a>controlling user agent</a>, but may be omitted if the
```

Comment:
s/may/MAY. Actually, I wonder if that should not be "MUST be implemented... unless", since I don't see a way not to implement the interface if the user agent supports monitoring.